dbms/tests/queries/0_stateless/01079_bit_operations_using_bitset.sql

```diff
@@ -0,0 +1,7 @@
+SELECT IPv6NumToString(bitAnd(IPv6StringToNum('2001:0db8:85a3:8d3a:b2da:8a2e:0370:7334'), IPv6StringToNum('ffff:ffff:ffff:0000:0000:0000:0000:0000'))) FROM system.numbers LIMIT 10;
```

Comment:
Better to implement also bitNot for completeness.

Reply:
done